android/Omnivore/app/src/main/java/app/omnivore/omnivore/ui/auth/EmailLogin.kt

```diff
@@ -46,7 +50,9 @@ fun EmailLoginView(viewModel: LoginViewModel, onAuthProviderButtonTap: () -> Uni
       }
 
       ClickableText(
-        text = AnnotatedString("Continue with Google/Apple ->"),
+        text = AnnotatedString("Return to Social Login"),
```

Comment:
Makes way more sense! Was struggling to think of something

Reply:
haha same. Technically we don't need anything since Android has a back button. Was also thinking just `Cancel`